src/clnt_vc.c

```diff
@@ -147,7 +147,7 @@ clnt_vc_ncreatef(const int fd,	/* open file descriptor */
 	struct x_vc_data *xd = NULL;
 	struct cx_data *cx = NULL;
 	struct ct_data *ct = NULL;
-	struct ct_serialized *cs = NULL;
+	struct ct_data *cs;
```

Comment:
Look 2 lines up: ct is already in use for the "other" ct_data instance -- and they are not the same (cs is per-client, ct is per-server). But the "other" goes away in a future patch, where its 2 used variables become merged with cx.

You'll note that all of the ct_serialized variables also start with ct_, even though they were not part of ct_data.

(Matt designed this. I'm just fixing it.)